library/modules/bigip_wait.py

```diff
@@ -23,6 +23,18 @@
     of rebooting due to a configuration change.
 version_added: 2.5
 options:
+  type:
+    description:
+      - The type of the BIGIP.
+      - Defaults to standard, the other choice being VCMP.
```

Comment:
should be C(standard) and C(vcmp), also do not capitalize parameters unless you really require them all in capital letters.

Reply:
Fixed. My explanation is a little more flowy. I hope that's fine.